### ISSUE TITLE
compute a large dataset for interpretability comparisons

### ISSUE THREAD

**Issue by peach-lucien** [ImperialCollegeLondon/hcga#36]
We can produce a large dataset with lots of different graph types: real, classic, random etc. We then compute all their features. For a given new graph we can output a set of similar graphs types in respect to the feature space. This will improve interpretability.

**Comment by arnaudon** [comment]
I guess we leave this for next paper?

**Comment by peach-lucien** [comment]
Yes this will be quite  big undertaking and may require a student working on it extensively.

**Comment by arnaudon** [comment]
we could have some more synthetic datasets as a first step towards this. I have put SBM with 1, 2, 3 and 4 clusters as a toy example.
